api/main_endpoints/routes/Messages.js

```diff
@@ -0,0 +1,101 @@
+const {
+  UNAUTHORIZED,
+  FORBIDDEN,
+  SERVER_ERROR
+} = require('../../util/constants').STATUS_CODES;
+const express = require('express');
+const router = express.Router();
+const bodyParser = require('body-parser');
+const User = require('../models/User.js');
+
+router.use(bodyParser.json());
+
+const clients = {};
+
+const writeMessage = ((roomId, message) => {
+  if (clients[roomId]) {
+    clients[roomId].forEach(client => client.res.write(`data: ${JSON.stringify(message)}\n\n`));
+  }
+});
+
+router.post('/send', async (req, res) => {
+
+  const {apiKey, message, id} = req.body;
+
+  let apiKeyFound = false;
+
+  if(!apiKey || !message || !id){
+    res.sendStatus(FORBIDDEN);
+    return;
+  }
+
+  await User.findOne({apiKey}, (error, result) => {
```

Comment:
we dont need to await if we are returning a response to the client inside the findOne callback